src/routes.js

```diff
@@ -0,0 +1,131 @@
+import 'babel-core/register';
```

Comment:
that import is useless. I thought I already removed them all, I'm going to purge this one two.